source/develop/release-management/features/storage/cinderlib-integration.html.md

```diff
@@ -24,13 +24,13 @@ This feature enables the user to be able to consume any storage backend supporte
 
 ### Detailed Description
 
-Cinder Library [1] provides a Python library that give the possibility to use the Cinder drivers directly without the need of a full Openstack/Cinder deployment supporting over 80 storage drivers.
+Cinder Library [1] provides a Python library that give the possibility to use the Cinder drivers directly without the need of a full OpenStack/Cinder deployment supporting over 80 storage drivers.
 
 - The user is able to configure a Storage Domain with the name of the Cinder driver and a set of parameters needed by the driver implementation.
-- The user is able to create/delete disks on that Storage Domain.
+- The user is able to create/delete/clone disks on that Storage Domain.
 - The user is able to create/delete snapshots of the disks created on that Storage Domain.
 - The user is able to run VMs with disks created on that Storage Domain.
-
+- The user is able to create templates from VMs with disks created on that Storage Domain.
```

Comment:
The user is able to create a VM from a template with disks on that storage domain

Reply:
Done